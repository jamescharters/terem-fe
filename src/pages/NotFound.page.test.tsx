import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import NotFound from "./NotFound.page";

describe("NotFound", () => {
    it("renders the component", () => {
        const result = render(<NotFound />);

        expect(result.asFragment()).toMatchSnapshot();
    });
});
