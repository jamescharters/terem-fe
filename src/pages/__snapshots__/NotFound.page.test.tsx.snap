// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound > renders the component 1`] = `
<DocumentFragment>
  <h1
    data-testid="page-heading"
  >
    404
  </h1>
  <p>
    You should not be here, bad things have happened.
  </p>
</DocumentFragment>
`;
